### DISCUSSION TITLE
FaceTime/WhatApp audio not coming through multi-output device

### DISCUSSION THREAD

**Discussion by jordanhowell7** [ExistentialAudio/BlackHole#734] [Q&A]
I've set up a simple multi-output device to capture my Macbook's audio through BlackHole and my Macbook's speakers at the same time, but it's not picking up audio from FaceTime or WhatsApp audio - which it was doing well yesterday.

Today I can only record it - which is does very quietly - but I cannot hear it through my speakers when it's recording.

It seems to be just be those apps not playing through my speakers.

I'm using Ventura 13.5 and have have made sure the speakers are at the top of the multi-output device list.

**Comment by jordanhowell7** [comment]
Here is a screenshot of my Audio MIDI Setup window, my FaceTime audio settings and a (quiet) audio recording of a FaceTime Audio call - which I can only hear through my speakers when I play it back.

![Screenshot 2023-08-19 at 11 10 42](https://github.com/ExistentialAudio/BlackHole/assets/99071994/94a3d2b8-7423-4ecd-9d8e-27c2106cb788)

**Comment by devinroth** [comment]
Are you sure it worked before because those app have been blocking BlackHole for a while.

Devin

**Comment by devinroth** [comment]
That being said, my new app Dipper will capture those without any problem.

https://apps.apple.com/app/dipper/id6450242673?mt=12

Devin

**Comment by PucaVaz** [comment]
Do you know how to fix that? I have  same issue
